src/index.ts

```diff
@@ -19,6 +24,13 @@ export default class BlueButton {
   version: string;
   baseUrl: string;
 
+  // AUTH props
+  usePKCE: boolean;
```

Comment:
Is there ever a reason to NOT use PKCE? Should we even let that be an option?

Reply:
@nbragdon That's right. We don't want to even give the option of not using it. I'll remove it.